packages/vite/src/node/cli.ts

```diff
@@ -212,8 +212,8 @@ cli
     `[string] build specified entry for server-side rendering`,
   )
   .option(
-    '--sourcemap',
-    `[boolean] output source maps for build (default: false)`,
+    '--sourcemap [mode]',
```

Comment:
nit: instead of `mode`, which conflicts with the `--mode` option, i think we could refer as `[output]` instead

```suggestion
    '--sourcemap [output]',
```